### PR TITLE
Adds more illegal/coin items to venders as well as redone its sorting to be better

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
1) I made all the vender lists not be eye death and impossible to read for coders that want to fine a time/add in
2) Add more rarer things into venders for coin items + high costs
3) Adds a lot more hacking items to venders + high costs

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1314,39 +1576,94 @@
...
 	vendor_department = DEPARTMENT_CHURCH
-	products = list(/obj/item/weapon/book/ritual/cruciform = 10, /obj/item/weapon/storage/fancy/candle_box = 10, /obj/item/weapon/reagent_containers/food/drinks/bottle/ntcahors = 20)
-	contraband = list(/obj/item/weapon/implant/core_implant/cruciform = 3)
-	prices = list(/obj/item/weapon/book/ritual/cruciform = 500, /obj/item/weapon/storage/fancy/candle_box = 200, /obj/item/weapon/reagent_containers/food/drinks/bottle/ntcahors = 250, /obj/item/weapon/implant/core_implant/cruciform = 1000)
+	products = list(/obj/item/weapon/book/ritual/cruciform = 10,
+				/obj/item/weapon/storage/fancy/candle_box = 10,
+				/obj/item/weapon/reagent_containers/food/drinks/bottle/ntcahors = 20)
+	contraband = list(/obj/item/weapon/implant/core_implant/cruciform = 3,
+				/obj/item/weapon/material/knife/neotritual = 3,
+				/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_old_guns = 3,
```
Comment (improvedname) [inline code/game/machinery/vending.dm]: Its a religious vendor not a gunvendor smh

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1314,39 +1576,94 @@
...
+				/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_new_guns = 3,)
```
Comment (improvedname) [inline code/game/machinery/vending.dm]: Its a religious vendor not a gunvendor smh

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1314,39 +1576,94 @@
...
+	prices = list(/obj/item/weapon/book/ritual/cruciform = 500,
+				/obj/item/weapon/storage/fancy/candle_box = 200,
+				/obj/item/weapon/reagent_containers/food/drinks/bottle/ntcahors = 250,
+				/obj/item/weapon/implant/core_implant/cruciform = 1000,
+				/obj/item/weapon/material/knife/neotritual = 750,
+				/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_old_guns = 4000,
```
Comment (improvedname) [inline code/game/machinery/vending.dm]: Its a religious vendor not a gunvendor smh

File: code/game/machinery/vending.dm [hunk 4]
```diff
@@ -1314,39 +1576,94 @@
...
+				/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_new_guns = 8000)
```
Comment (improvedname) [inline code/game/machinery/vending.dm]: Its a religious vendor not a gunvendor smh

File: code/game/machinery/vending.dm [hunk 5]
```diff
@@ -1266,45 +1473,100 @@
...
+				/obj/item/clothing/gloves/insulated = 600,
+				/obj/item/weapon/tool/screwdriver = 30,
+				/obj/item/weapon/tool/crowbar = 30,
+				/obj/item/weapon/tool/wirecutters = 30,
+				/obj/item/weapon/tool/multitool = 40,
+				/obj/item/weapon/tool/wrench = 40,
+				/obj/item/device/t_scanner = 50,
+				/obj/item/weapon/cell/large = 500,
+				/obj/item/weapon/cell/small = 100,
+				/obj/item/weapon/cell/medium = 300,
```
Comment (improvedname) [inline code/game/machinery/vending.dm]: why batteries for the engi vendor? theres a special asters battery vendor for this

Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: The large cells were already in their, added the other ones that costs more.

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: The guns are religious

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: The guns are religious

File: code/game/machinery/vending.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: The guns are religious - This one prints basically worst tech then void worrier so its costing the same

File: code/game/machinery/vending.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: The guns are religious, this one being really strong so it costs more. They DO crusade against Lings and other baddies ~~like IH~~

File: code/game/machinery/vending.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/vending.dm]: OH FOR TOOLS

**Comment by improvedname** [comment]
Coin slot vendor items are stupid and shouldnt exist, its old code used for a makeshift economy. But we have an auctall economy now. So coin slots are not needed.
Also that high cells for engivendor as contrabrand is stupid stuff i already told you there is a whole vendor for buying cells. This is just conviencecreep

**Comment by PolymorphBlue** [comment]
Personally, I'm okay with all of it.  But I've never really minded convienencecreep.  Anyone else care to weigh in?

**Comment by StellarTheAwoo** [comment]
More additions to the contraband section of vendors are always welcome, especially for the neotheology vendor. In the end though we will most likely be scrapping neotheology when our new lore/map comes around, but in the mean time, we could always impose contraband lawsets for having these disks if needed.

**Comment by Trilbyspaceclone** [comment]
Removes water flower from seed venders

**Comment by rpilcrow** [comment]
can we get an exact list of what got added?
and i agree with improvedname - the premium list is basically defunct now that there are actual price settings for vendor items.

**Comment by Trilbyspaceclone** [comment]
Meh

**Pull request closed without merging**